src/main/app/src/components/useKoodistoOptions/index.js

```diff
@@ -19,7 +19,11 @@ const getOptions = ({ koodisto, language, sort = true }) => {
     label: getFirstLanguageValue(nimi, language),
   }));
 
-  return sort ? sortBy(options, ({ label }) => label) : options;
+  const byLabel = ({ label }) => label;
+  const byFirstNumber = ({ label }) =>
+    /^\d/.test(label) && parseInt(first(label.match(/(\d+)/)));
```

Comment:
Tämä sorttaus ei taida toimia ihan oikein, jos on sekaisin labeleita, jotka alkaa numerolla ja jotka ei. ByFirstNumber-funktion pitää palauttaa undefined muille kuin numeroilla alkaville labeleille, jotta niitä ei sortata. Muutenkin hieman parempi voisi olla käyttää lodashin toNumber:a.
```suggestion
    toNumber(first(label.match(/(^\d+)/)));
```